ghost/admin/app/components/posts-list/context-menu.js

```diff
@@ -15,10 +19,36 @@ export default class PostsContextMenu extends Component {
     }
 
     @action
-    deletePosts() {
-        // Use filter in menu.selectionList.filter
-        alert('Deleting posts not yet supported.');
+    async deletePosts() {
         this.menu.close();
+        const confirmedDelete = await this.modals.open(DeletePostsModal, {
+            isSingle: this.selectionList.isSingle,
+            confirm: this.deletePostsTask
+        });
+        if (!confirmedDelete) {
+            return;
```

Comment:
I don't understand this block - why do we need to do this?

Reply:
Ah, good catch. This was a leftover before I made the modal async so that it has a progress button while deleting the posts. I'll remove this part